### ISSUE TITLE
A-C 60.0.6 release fails with beetmover errors

### ISSUE THREAD

**Issue by st3fan** [mozilla-mobile/android-components#8834]
The following run https://github.com/mozilla-mobile/android-components/runs/1322850726 with task group https://firefox-ci-tc.services.mozilla.com/tasks/groups/VqVIRjf1T0OiBmLD30swCA is failing on all the `beetmover-*` tasks with the following error:

```
2020-10-28 20:02:19,017 - beetmoverscript.task - INFO - Action types: ['push-to-maven']
2020-10-28 20:02:19,018 - scriptworker.client - DEBUG - Task is validated against this schema: {'title': 'Taskcluster beetmover task maven schema', 'type': 'object', 'properties': {'dependencies': {'type': 'array', 'minItems': 1, 'uniqueItems': True, 'items': {'type': 'string'}}, 'payload': {'type': 'object', 'properties': {'upload_date': {'type': 'number'}, 'build_number': {'type': 'number'}, 'locale': {'type': 'string'}, 'maxRunTime': {'type': 'number'}, 'version': {'type': 'string'}, 'next_version': {'$comment': 'ignored'}, 'appVersion': {'$comment': 'ignored'}, 'releaseProperties': {'type': 'object', 'properties': {'appName': {'type': 'string'}, 'buildid': {'type': 'string'}, 'appVersion': {'type': 'string'}, 'hashType': {'type': 'string'}, 'platform': {'type': 'string'}, 'branch': {'type': 'string'}}, 'required': ['appName']}, 'upstreamArtifacts': {'type': 'array', 'items': {'type': 'object', 'properties': {'taskType': {'type': 'string'}, 'taskId': {'type': 'string'}, 'paths': {'type': 'array', 'minItems': 1, 'uniqueItems': True, 'items': {'type': 'string'}}}, 'required': ['taskId', 'taskType', 'paths']}, 'minItems': 1, 'uniqueItems': True}}, 'required': ['upstreamArtifacts', 'releaseProperties']}}, 'required': ['payload', 'dependencies']}
2020-10-28 20:02:19,020 - beetmoverscript.task - INFO - Buckets: ['maven-production']
2020-10-28 20:02:19,021 - beetmoverscript.task - INFO - Action types: ['push-to-maven']
2020-10-28 20:02:19,021 - beetmoverscript.task - DEBUG - Loading release_props from task's payload: {'appName': 'components'}
Traceback (most recent call last):
  File "/app/bin/beetmoverscript", line 8, in <module>
    sys.exit(main())
  File "/app/lib/python3.8/site-packages/beetmoverscript/script.py", line 584, in main
    client.sync_main(async_main, config_path=config_path, default_config=default_config, should_validate_task=False)
  File "/app/lib/python3.8/site-packages/scriptworker/client.py", line 166, in sync_main
    loop.run_until_complete(_handle_asyncio_loop(async_main, context))
  File "/usr/local/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/app/lib/python3.8/site-packages/scriptworker/client.py", line 205, in _handle_asyncio_loop
    await async_main(context)
  File "/app/lib/python3.8/site-packages/beetmoverscript/script.py", line 292, in async_main
    await action_map[context.action](context)
  File "/app/lib/python3.8/site-packages/beetmoverscript/script.py", line 212, in push_to_maven
    version = task.get_maven_version(context)
  File "/app/lib/python3.8/site-packages/beetmoverscript/task.py", line 89, in get_maven_version
    version = context.task["payload"]["version"]
KeyError: 'version'
exit code: 1
```

**Comment by st3fan** [comment]
@mozilla-mobile/releng help please - this is a release blocker

**Comment by st3fan** [comment]
This may be fixed with https://github.com/mozilla-mobile/android-components/pull/8843 which was just landed on the releases/60.0 branch.

**Comment by escapewindow** [comment]
@st3fan still an issue?
